packages/react-data-grid/src/masks/InteractionMasks.js

```diff
@@ -646,9 +646,9 @@ class InteractionMasks extends React.Component {
     this.copyMask = node;
   };
 
-  getSelectedDimensions = (selectedPosition) => {
-    const { scrollLeft, getRowHeight, getRowTop, getRowColumns } = this.props;
-    const columns = getRowColumns(selectedPosition.rowIdx);
+  getSelectedDimensions = (selectedPosition, useGridColumns) => {
+    const { scrollLeft, getRowHeight, getRowTop, getRowColumns, columns: gridColumns } = this.props;
+    const columns = useGridColumns ? gridColumns : getRowColumns(selectedPosition.rowIdx);
```

Comment:
Not familiar with the RDG codebase yet, but why not just always use `this.props.columns`? Is there a benefit to using `getRowColumns()` in this method?

Reply:
I am not 100% sure but I think `getRowColumns`, `getRowTop` methods were added to support variable height rows and grouping. I found [this bug](https://github.com/adazzle/react-data-grid/pull/1289/files) that I fixed but I do not remember the details (always fill in ticket details)

It is probably the same issue where you are copying values form a row with less columns

Reply:
I think we may be able to remove `getRowColumns` as I fixed a different issue with copy mask
https://github.com/adazzle/react-data-grid/pull/1289/files#diff-04fdd57267f50e49721eb930c70e4059R640